### PR TITLE
New Good quality code Commands. Working with current index.js/command handler

### PR DESCRIPTION
**Pull request by ManavvGarg**
New commands submitted again. Good quality code, working with current index.js.

Added a reload command to minimize stopping the whole bot and instead reload a specific command.

### CODE REVIEW

File: examples/commands/reload.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Or copy the code inside the run function as its simply the message and arguments.
+*/
+
+
+const Discord = require("discord.js")
+
+module.exports = {
+    name: "reload",
+    run: async (client, message, args) => {
```
Comment (Anish-Shobith) [inline examples/commands/reload.js:16]: Fix the function argument , `client` is a parameter here instead use `message.client`

File: examples/commands/reload.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        let embed = new Discord.MessageEmbed()
```
Comment (Anish-Shobith) [inline examples/commands/reload.js:18]: Lets use `const` as much as possible.

File: examples/commands/reload.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        .setTitle("Reload")
+        .setDescription("Sorry, the `reload` command can only be executed by the Developer.")
+        .setColor("#cdf785");
+        if(message.author.id !== 'YOUR DISCORD ID') return message.channel.send(embed);
+
+        if(!args[0].toLowerCase()) return message.channel.send("Please provide a command name!")
+
+        let commandName = args[0].toLowerCase()
```
Comment (Anish-Shobith) [inline examples/commands/reload.js:26]: Same above `const`

File: examples/commands/reload.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+        try {
+          
+          delete require.cache[require.resolve(`./${commandName}.js`)]
```
Comment (Anish-Shobith) [inline examples/commands/reload.js:30]: Looks good to me but lets make sure the command exists or not and make sure its removed the collection too and then added again
`message.client.commands.delete(key)`;

File: examples/commands/remove.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+module.exports = {
+        
+	name: 'remove',
+	run: async(message, args) => {
+
+		const player = message.client.manager.players.get(message.guild.id);
+
+		if (isNaN(args[0])) return message.channel.send('Invalid number.');
+
+        if (args[0] == 0) return message.channel.send(`Cannot remove a song that is already playing.`);
```
Comment (Anish-Shobith) [inline examples/commands/remove.js:21]: use `===` instead of `==`

File: examples/commands/remove.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        if (args[0] > player.queue.length) return message.channel.send('Song not found.');
+
+        const { title } = player.queue[args[0] - 1];
+
+        player.queue.splice(args[0] - 1, 1);
+        return message.channel.send(`Removed ***${title}*** from the queue`);
```
Comment (Anish-Shobith) [inline examples/commands/remove.js:27]: Well sometimes its annoying when markdown is not parsed properly , use the [`escapeMarkdown`](https://discord.js.org/#/docs/main/stable/class/Util?scrollTo=s-escapeMarkdown) from the discord.js [`Utils`](https://discord.js.org/#/docs/main/stable/class/Util) class. Its a static method so no need to invoke it. Just pass the title through the title and then use the markdown

File: examples/commands/seek.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+		
+		run: async (message, args) => {
+
+		if(isNaN(args[0]) && args[0] < 0) return message.reply(`Invalid number. Please provide a number in seconds.\nCorrect Usage: \`seek <seconds>\``);
+
+		const player = message.client.manager.players.get(message.guild.id);
+		if(args[0] * 1000 >= player.queue.current.length || args[0] < 0) return message.channel.send('Cannot seek beyond length of song.');
+		player.seek(args[0] * 1000);
+
+		return message.channel.send(`Seeked ${args[0]} second(s)`);
```
Comment (Anish-Shobith) [inline examples/commands/seek.js:22]: Would be cleaner if you make a function for checking seconds aka plural

File: examples/commands/shuffle.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+Or copy the code inside the run function as its simply the message and arguments.
+*/
+
+
+
+module.exports = {
+    
+    name: "shuffle",
+    run: async(bot, message, args) => {
```
Comment (Anish-Shobith) [inline examples/commands/shuffle.js:16]: No parameter bot here, just `message` and `args`

File: examples/commands/jump.js [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+            if (args[0] === 0) return message.channel.send(`Cannot skip to a song that is already playing.`);
+    
+            const player = message.client.manager.players.get(message.guild.id);
+            if ((args[0] > player.queue.length) || (args[0] && !player.queue[args[0] - 1])) return message.channel.send('Song not found.');
+            const { title } = player.queue[args[0] - 1];
+            if (args[0] == 1) player.stop();
+            player.queue.splice(0, args[0] - 1);
+            player.stop();
+    
+            return message.channel.send(`Skipped to **${title}**.`);
```
Comment (Anish-Shobith) [inline examples/commands/jump.js:27]: Same thing as the Utils class thing , down below.

**Review by Anish-Shobith — CHANGES_REQUESTED** [review]

File: examples/commands/jump.js (hunk 9 above)
Comment (karelkryda) [reply examples/commands/jump.js:27]: You can use `player.queue.remove(0, args[0] - 1);`

**Review by Strandxo — COMMENTED** [review]
This entire commit seems to be really unnecessary... Solaris provides some basic examples to go off... you're providing an entire bot here...

**Comment by HysMX** [comment]
> This entire commit seems to be really unnecessary... Solaris provides some basic examples to go off... you're providing an entire bot here...

Some people struggle with the current provided example, this solution would give users another view of how to do stuff

**Comment by Strandxo** [comment]
> > This entire commit seems to be really unnecessary... Solaris provides some basic examples to go off... you're providing an entire bot here...
> 
> Some people struggle with the current provided example, this solution would give users another view of how to do stuff

This seems like the biggest load of cr*p. The current examples are completely fine. You're adding additional code which isn't necessary. Simply add in line comments to the code to explain it instead of adding more code which isn't explained.

**Comment by ManavvGarg** [comment]
> > > This entire commit seems to be really unnecessary... Solaris provides some basic examples to go off... you're providing an entire bot here...
> > 
> > 
> > Some people struggle with the current provided example, this solution would give users another view of how to do stuff
> 
> This seems like the biggest load of cr*p. The current examples are completely fine. You're adding additional code which isn't necessary. Simply add in line comments to the code to explain it instead of adding more code which isn't explained.

Just because you dont want the code to be open sourced does not mean you have to stop others. I know you dont like the code to be open sourced and that you should not be on github. Why create a ruckus over this small thing? Are you an idiot or something? Cause i would love to not give crap about you. Have a good day kid.

**Comment by ManavvGarg** [comment]
Imma take my time, revamp the code and explain it properly, with comments. Also I will start using consts from now own. @Anish-Shobith

**Pull request closed without merging**